index.js

```diff
@@ -68,6 +68,42 @@ class CreatBook {
     bookContainer.appendChild(bookDiv);
   }
 
+  static openListPage() {
+    const bookList = document.getElementById('list');
+    const listSection = document.querySelector('.list-section');
+    const addSection = document.querySelector('.add-section');
+    const contactSection = document.querySelector('.contact-section');
+    bookList.addEventListener('click', () => {
+      addSection.classList.add('hidden');
+      listSection.classList.remove('hidden');
+      contactSection.classList.add('hidden');
+    });
+  }
+
+  static openAddPage() {
+    const bookList = document.getElementById('add');
+    const listSection = document.querySelector('.list-section');
+    const addSection = document.querySelector('.add-section');
+    const contactSection = document.querySelector('.contact-section');
+    bookList.addEventListener('click', () => {
+      addSection.classList.remove('hidden');
+      listSection.classList.add('hidden');
+      contactSection.classList.add('hidden');
+    });
+  }
+
+  static openContactPage() {
+    const bookList = document.getElementById('contact');
+    const listSection = document.querySelector('.list-section');
```

Comment:
- [ ] [OPTIONAL] Don't you think it will be better if you created a reusable function for these methods. Kindly look into it